### DISCUSSION TITLE
up主个人空间内偶尔出现访问权限受限提示

### DISCUSSION THREAD

**Discussion by Shaddoll126** [ywmoyue/biliuwp-lite#107] [General]
就有时候会出现投稿视频不能加载,弹出提示:访问权限受限制
大概在进入up主个人空间几分钟后就会有概率出现  不知道是不是bug所以在这边反应一下:3

**Comment by ywmoyue** [comment]
什么版本，麻烦把日志通过邮箱发送一下

邮箱地址在关于页能找到

**Comment by ywmoyue** [comment]
了解了，接口参数有调整，参考此issue：https://github.com/SocialSisterYi/bilibili-API-collect/issues/631

我有空修复一下

**Comment by Shaddoll126** [comment]
好耶

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
